packages/store-sync/src/recs/encodeEntity.ts

```diff
@@ -0,0 +1,19 @@
+import { Entity } from "@latticexyz/recs";
+import { StaticAbiType } from "@latticexyz/schema-type";
+import { encodeAbiParameters } from "viem";
+import { SchemaToPrimitives } from "../common";
+import { hexKeyTupleToEntity } from "./hexKeyTupleToEntity";
+
+export function encodeEntity<TKeySchema extends Record<string, StaticAbiType>>(
```

Comment:
I was gonna move these into RECS but it's a bigger refactor and we have to figure out where to draw lines in terms of type definitions (where should `KeySchema` and `ValueSchema` live and, if not centralized, is it okay to duplicate them in a few packages?)

Reply:
After thinking more about this I think it makes sense to keep these utils in here instead of in `recs`. In `recs` land the entity is just a string, while the encoding and decoding based on key schema is a concept on top of it.